LPS-181595/LPS-176185 Uniform Automated Test | Automation for LPS-181594 - New Property - DL size timeout

lucasperj opened the pull request:
Issue link: [LPS-181595](https://issues.liferay.com/browse/LPS-181595)
Story related [ticket](https://issues.liferay.com/browse/LPS-181594) and [bug](https://issues.liferay.com/browse/LPS-176185)
Related backend PR: https://github.com/liferay-uniform/liferay-portal/pull/1111

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

lucasperj commented:
> @lucasperj We will need to clean this up tomorrow, I know we need to finish by end of week, but this is not ready yet.
> 
> 1. What is the difference between UpgradeReportFilePathMessageLogged and the existing test? https://github.com/liferay/liferay-portal/blob/f37b1ce60e91915c4b3b60abcd88c47acac7c9cb/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase#L512. The new test is checking the upgrade report to tell you where the upgrade report is? That doesn't make sense to me, why does the user need that info?
> The current test is worded "Given A upgrade is completed When Upgrade Report is enabled Then a message saying "Starting upgrade report generation" is printed right after the Upgrade logs" - what file is this printed to?
> They are so similar, the new test was actually deleted, maybe by SF?
>  
> 2. Once 1 is resolved, I also noticed that 2 new tests could probably be combined, we check the user gets notified of a report getting generated, and then check the user gets notified about the report location. These feel fine to combine in Poshi, especially if we are checking the console log where the user needs to see this information.
>  
> 3. I also want you to try using task descriptions that use the "Given, When, Then" pattern that is focused on the user. Just apply to this new test for now, we can think about applying to the rest of these testcase later.

@vicnate5 

1. UpgradeReportFilePathMessageLogged is using the DB Upgrade Tool, that prior to these changes did not tell the user the way the report was generated. In the existing test, we used Auto Upgrade, which generates the report in a diff path, in the liferay home, not inside portal-tools-db-upgrade-client, I've changed the names to try to make clear the diff , the old test now is `ReportTraceLineLoggedWithAutoUpgrade` and the new `ReportTraceLineLoggedWithDBUpgradeTool`
2. I've combined both now 
3. Added the tasks descriptions with the "Given, When, Then" scenarios

lucasperj commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-source-format/4766/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/4766/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/4766/">test-portal-source-format#4766</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/4766/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/vicnate5/liferay-portal/pull/2309">vicnate5#2309</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2275809279">[master] ci:test:sf - lucasperj &gt; vicnate5 - PR#2309 - 2023-04-20[07:58:04]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/17787/">publish-testray-report#17787</a><br/></div></div>

vicnate5 commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/133738

See my changes.
